sopel/config/core_section.py

```diff
@@ -216,3 +216,9 @@ def homedir(self):
 
     verify_ssl = ValidatedAttribute('verify_ssl', bool, default=True)
     """Whether to require a trusted SSL certificate for SSL connections."""
+
+    auto_url_schemes = ListAttribute(
+        'auto_url_schemes',
+        strip=True,
+        default=('http', 'https', 'ftp'))
```

Comment:
`ListAttribute` should default to a `list`, hmm?
```suggestion
        default=['http', 'https', 'ftp'])
```